prismic-model/js/model/thumbnails.js

```diff
@@ -0,0 +1,9 @@
+// @flow
+
+type Thumbnail = {
+  name: string,
+  width: number,
+  height: number,
+};
+
+export type Thumbnails = Array<Thumbnail>;
```

Comment:
nit: Sorry to be a pedant here, and sorry for not being clear, but we would normally just use `Array<Thumbnail>` where we need it rather than create a new type for it.

Reply:
with you .. thanks for that